src/cljs/main/broadfcui/common/style.cljs

```diff
@@ -101,7 +101,7 @@
              {} props))
 
 (defn create-text-field [props]
-  [:input (utils/deep-merge {:type "text" :style input-text-style} (replace-nil-value-key props))])
+  [:input (utils/deep-merge {:type (or (:type props) "text") :style input-text-style} (replace-nil-value-key props))])
```

Comment:
this, and the associated change in input.cljs, allow callers of `TextField` to specify the DOM type for the input. In this PR's use case, it allows us to specify `password` instead of `text`.